qiita_db/support_files/patches/39.sql

```diff
@@ -0,0 +1,15 @@
+-- Sep 7, 2016
+-- Adding a is numeric function to avoid problems with texts
+-- http://stackoverflow.com/a/16206123
+
+CREATE OR REPLACE FUNCTION isnumeric(text) RETURNS BOOLEAN AS $$
```

Comment:
Can you add a test for this function in here? https://github.com/biocore/qiita/blob/master/qiita_db/test/test_sql.py